tests/unit/test_initialize.py

```diff
@@ -169,11 +168,6 @@ def test__initialize__token_missing() -> None:
             kolena.initialize()
 
 
-def test__uninitialized_usage() -> None:
-    with pytest.raises(UninitializedError):
-        kolena.fr.Model.create("test", {})
```

Comment:
We should still test this -- it's functionality that's relevant still.

Reply:
Added back